Octokit.Tests.Integration/Clients/PullRequestsClientTests.cs

```diff
@@ -59,6 +59,7 @@ public PullRequestsClientTests()
 
         Assert.Equal(1, pullRequests.Count);
         Assert.Equal(result.Title, pullRequests[0].Title);
+        Assert.NotNull(pullRequests[0].Id);
```

Comment:
Thanks for the catch. I am going to update that. I am assuming all internal ids  are larger than 0.